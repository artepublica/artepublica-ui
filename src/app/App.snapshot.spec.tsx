import { create } from 'react-test-renderer';

import App from './App';

describe('<App />', () => {
    it('default snapshot', () => {
        const json = create(<App />).toJSON();

        expect(json).toMatchSnapshot();
    });
});
