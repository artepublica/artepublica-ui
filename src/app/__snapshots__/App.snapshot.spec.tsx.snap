// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> default snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": undefined,
      "flex": 1,
      "overflow": "hidden",
    }
  }
/>
`;
